.changeset/cuddly-peas-try.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/react-brand": patch
```

Comment:
This is technically a breaking change, so can we change this to `minor` please? (Lib is pre-v1, so we don't need to change major yet)

Reply:
```suggestion
"@primer/react-brand": minor
```